src/privateJrPlatform/getJrPlatformApp.ts

```diff
@@ -0,0 +1,40 @@
+import { load, Obj } from 'scrivito'
+import { App } from '../App'
+import { WrongContentFormat } from './Components/WrongContentFormat'
+
+export async function getJrPlatformApp() {
+  return (await isValidContentFormat()) ? App : WrongContentFormat
+}
+
+const CONTENT_FORMAT = 'portal-app:5'
+const KNOWN_CONTENT_FORMATS: Record<string, string> = {
+  'portal-app:5': 'https://scrivito-portal-app.pages.dev',
+
+  /* Once v6 is released, the list of KNOWN_CONTENT_FORMATS looks like this: */
+  // 'portal-app:5': 'https://v5.scrivito-portal-app.pages.dev',
+  // 'portal-app:6': 'https://scrivito-portal-app.pages.dev',
+}
```

Comment:
* [ ] FYI From the comment it's not 100% clear if a v5 app needs an update when/after/before v6 comes out. Maybe the comment can make clearer, which versions need which updates on a new content version.

Reply:
`KNOWN_CONTENT_FORMATS` should be identical between `v5` and `v6`.

When `v6` get's released for the first time it can be deployed first and `v5` soon after.